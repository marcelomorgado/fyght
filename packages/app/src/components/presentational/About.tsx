import React from "react";
import { Card } from "antd";

export const About = () => {
  return (
    <Card hoverable type="inner" title="About & Rules">
      <p>Your journey to became a master starts now.</p>
      <p>
        Train with your master and challenge another players to see who is the
        best.
      </p>
      <p>After each victory, your fighter gains +1 XP and become stronger.</p>
    </Card>
  );
};

export default About;
