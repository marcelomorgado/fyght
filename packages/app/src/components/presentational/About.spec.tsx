import * as React from "react";
import { shallow } from "enzyme";

import { About } from "./About";

describe("About", () => {
  test("should render the component", () => {
    expect(shallow(<About />)).toMatchSnapshot();
  });
});
