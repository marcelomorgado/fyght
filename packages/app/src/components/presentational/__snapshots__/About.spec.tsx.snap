// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render the component 1`] = `
<Card
  title="About & Rules"
  type="inner"
>
  <p>
    Your journey to became a master starts now.
  </p>
  <p>
    Train with your master and challenge another players to see who is the best.
  </p>
  <p>
    After each victory, your fighter gains +1 XP and become stronger.
  </p>
</Card>
`;
